### PR TITLE
Release v2.1.1-rc1

### PR DESCRIPTION
**Pull request by ebfull**

### CODE REVIEW

File: doc/man/zcashd.1 [hunk 1]
```diff
@@ -67,15 +67,9 @@ Imports blocks from external blk000??.dat file on startup
...
-\fB\-mempooltxinputlimit=\fR<n>
-.IP
-[DEPRECATED FROM OVERWINTER] Set the maximum number of transparent
-inputs in a transaction that the mempool will accept (default: 0 = no
-limit applied)
-.HP
 \fB\-par=\fR<n>
 .IP
-Set the number of script verification threads (\fB\-2\fR to 16, 0 = auto, <0 =
+Set the number of script verification threads (\fB\-16\fR to 16, 0 = auto, <0 =
```
Comment (str4d) [inline doc/man/zcashd.1]: Lol, @ebfull can haz many cores

File: src/deprecation.h [hunk 2]
```diff
@@ -8,7 +8,7 @@
 // Deprecation policy:
 // * Shut down 16 weeks' worth of blocks after the estimated release block height.
 // * A warning is shown during the 2 weeks' worth of blocks prior to shut down.
-static const int APPROX_RELEASE_HEIGHT = 634650;
+static const int APPROX_RELEASE_HEIGHT = 684782;
```
Comment (str4d) [inline src/deprecation.h]: A little further back than expected, but it's fine.

File: doc/man/zcashd.1 (hunk 1 above)
Comment (ebfull) [reply doc/man/zcashd.1]: haha sorry

**Comment by ebfull** [comment]
@zkbot r+

File: doc/man/zcashd.1 [hunk 3]
```diff
@@ -364,16 +360,24 @@ Prepend debug output with timestamp (default: 1)
...
 \fB\-printtoconsole\fR
 .IP
 Send trace/debug info to console instead of debug.log file
+.PP
+Chain selection options:
 .HP
 \fB\-testnet\fR
 .IP
-Use the test network
+Use the test chain
```
Comment (daira) [inline doc/man/zcashd.1]: Huh, when did upstream change this? "Test network" is the term that the spec uses (and in fact it is a separate network as well as a separate chain). Oh well, I guess it's not important.

File: src/deprecation.h [hunk 4]
```diff
@@ -8,7 +8,7 @@
...
 static const int WEEKS_UNTIL_DEPRECATION = 16;
 // static const int DEPRECATION_HEIGHT = APPROX_RELEASE_HEIGHT + (WEEKS_UNTIL_DEPRECATION * 7 * 24 * 24);
```
Comment (daira) [inline src/deprecation.h]: This should be reinstated. As it is now, `DEPRECATION_HEIGHT` stays at 734240.

**Comment by daira** [comment]
https://github.com/zcash/zcash/pull/4289

File: src/deprecation.h (hunk 4 above)
Comment (daira) [reply src/deprecation.h]: https://github.com/zcash/zcash/pull/4289

**Pull request merged**